src/Sulu/Bundle/ContentBundle/Resources/public/js/components/resource-locator/main.js

```diff
@@ -81,7 +81,7 @@ define(['config'], function(Config) {
             }.bind(this));
 
             // value change
-            this.sandbox.dom.on(getId.call(this, 'input'), 'change', setDataValue.bind(this));
+            this.sandbox.dom.on(getId.call(this, 'input'), 'keyup', setDataValue.bind(this));
```

Comment:
@danrot the `change` sometimes (on new pages) does not detect the change of the input.